pandas/tseries/period.py

```diff
@@ -530,6 +532,11 @@ def shift(self, n):
         values[mask] = tslib.iNaT
         return PeriodIndex(data=values, name=self.name, freq=self.freq)
 
+    @cache_readonly
+    def dtype_str(self):
+        """ return the dtype str of the underlying data """
+        return self.inferred_type
```

Comment:
when we add a real period dtype then can fix this
can u add a xref to that issue (point to this PR)